src/main/config/messages_template.xml

```diff
@@ -808,6 +808,11 @@ See the accompanying LICENSE file for applicable license.
     <reason>A content reference in a constrained document type cannot be resolved because it would violate one of the document constraints "%1".</reason>
     <response>The current constrained document may only reuse content from documents with equivalent constraints.</response>
   </message>
+  
+  <message id="DOTX077D" type="DEBUG">
```

Comment:
@robander @infotexture Should this be info level instead? Is this something that you care about? With content that has a lot of duplicate `@id` in resulting conreffed content, this can cause a lot of messages.

Reply:
I think this should be INFO level, could even say warning. For most sources this will not come up, but when it does it has the potential to change how your links resolve.

Reply:
Converted this to info.